### ISSUE TITLE
out-of-topic: How does fzf can still show tui while in subprocess?

### ISSUE THREAD

**Issue by leiless** [junegunn/fzf#3005]
Hi, @junegunn and `fzf` developers.

Thank you all for inventing this great tool, I've used this tool in my daily life.
Recently I want to use the terminal UI to implement a utility.
But I found that `fzf` can actually still show the terminal UI while in a subprocess call (so that the result can be used elsewhere). e.g. `$(fzf) | xargs ls -l`

All the details are described in the following issue:
https://github.com/fdehau/tui-rs/issues/674

I'm using Rust [tui-rs](https://github.com/fdehau/tui-rs) as my terminal UI backend, I guess this issue is not relevant to `fzf` itself, but I'm very glad if anyone can help me.

Basically, the `tui-rs` process cannot show terminal UI in a subprocess call, I have no previous background in terminal programming, and I don't even know how to exactly describe the issue I met.

![](https://user-images.githubusercontent.com/38041294/194698920-c48abe0a-49f4-47b7-9e02-03350e1f5454.gif)

**Comment by Delapouite** [comment]
Hi.

I think the trick used by Fzf is to display the UI using `stderr` instead of `stdout` as explained here : https://github.com/junegunn/fzf/discussions/2686

**Comment by leiless** [comment]
> Hi.
> 
> I think the trick used by Fzf is to display the UI using `stderr` instead of `stdout` as explained here : #2686

Thank you! I found that very helpful. And I posted a question in #2686. Can you help me with that?
